src/stripe.common.ts

```diff
@@ -53,11 +53,10 @@ Stripe.HttpClient = HttpClient.HttpClient;
 Stripe.HttpClientResponse = HttpClient.HttpClientResponse;
 
 import CryptoProvider = require('./crypto/CryptoProvider');
-import EventEmitter = require('events');
 Stripe.CryptoProvider = CryptoProvider;
 
-import DefaultPlatformFunctions = require('./platform/DefaultPlatformFunctions');
-Stripe._platformFunctions = new DefaultPlatformFunctions();
+import PlatformFunctions = require('./platform/PlatformFunctions');
+Stripe._platformFunctions = new PlatformFunctions();
```

Comment:
Do we need this? `PlatformFunctions` should never be instantiated. It's a "broken" implementation.